logger.go

```diff
@@ -0,0 +1,34 @@
+package cron
+
+import "log"
+
+// Logger is the simplest interface for logging
+type Logger interface {
+	Logf(format string, args ...interface{})
+}
+
+// NewStandartLogger returns a "Logger" wrapper for the standard go-logger;
+// it should help to migrate to the interface
+func NewStandartLogger(logger *log.Logger) *StandardLogger {
```

Comment:
Where is `NewStandardLogger` being used? 
btw, there is a typo in `NewStandartLogger`.